pkgs/development/python-modules/google-cloud-datastore/default.nix

```diff
@@ -3,30 +3,44 @@
 , fetchPypi
 , google-api-core
 , google-cloud-core
+, google-cloud-testutils
 , libcst
-, proto-plus
 , mock
-, pytestCheckHook
+, proto-plus
 , pytest-asyncio
-, google-cloud-testutils
+, pytestCheckHook
+, pythonOlder
 }:
 
 buildPythonPackage rec {
   pname = "google-cloud-datastore";
   version = "2.1.0";
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "1yyk9ix1jms5q4kk76cfxzy42wzzyl5qladdswjy5l0pg6iypr8i";
   };
 
-  propagatedBuildInputs = [ google-api-core google-cloud-core libcst proto-plus ];
+  propagatedBuildInputs = [
+    google-api-core
+    google-cloud-core
+    libcst
+    proto-plus
+  ];
 
-  checkInputs = [ google-cloud-testutils mock pytestCheckHook pytest-asyncio ];
+  checkInputs = [
+    google-cloud-testutils
+    mock
+    pytestCheckHook
+    pytest-asyncio
+  ];
 
   preCheck = ''
     # directory shadows imports
     rm -r google
+    ls
+    echo "------------------------------------------*************************************"
```

Comment:
Is this left from debuggin?

Reply:
Yes, that file shouldn't be part of the PR. Thanks for checking.